### PR TITLE
docs: crd install skip flag

### PR DESCRIPTION
**Pull request by balaharish7**

### CODE REVIEW

**Comment by avishnu** [comment]
Thanks for the change. Need to also document the options to skip installation of lvm/zfs local pv. Can be done in a separate pull request.

File: docs/main/quickstart-guide/installation.md [hunk 1]
```diff
@@ -81,11 +81,17 @@ helm install openebs --namespace openebs openebs/openebs --create-namespace
...
+- The default OpenEBS helm chart will install both Local Storage and Replicated Storage. If you do not want to install OpenEBS Replicated Storage, use the following command:
 
-```
-helm install openebs --namespace openebs openebs/openebs --set engines.replicated.mayastor.enabled=false --create-namespace
-```
+  ```
+  helm install openebs --namespace openebs openebs/openebs --set engines.replicated.mayastor.enabled=false --create-namespace
+  ```
+
+- If the CustomResourceDefinitions for CSI VolumeSnapshots are already present in your cluster, you may skip their creation by using the following command:
```
Comment (tiagolobocastro) [inline docs/main/quickstart-guide/installation.md:90]: Should we also have some information somewhere, stating that the snapshot CRDs are not uninstalled? CC @niladrih

Comment (niladrih) [inline docs/main/quickstart-guide/installation.md:90]: This line isn't a command in itself. Might be more appropriate to end with 'using the following option'.

Comment (niladrih) [reply docs/main/quickstart-guide/installation.md:90]: Sure. We could add in a new docs section for 'Uninstallation'. Maybe add a footnote to the 'Installation' page with info on uninstallation. Unsure of what would be best. Also tagging @avishnu and @Abhinandan-Purkait

Comment (balaharish7) [reply docs/main/quickstart-guide/installation.md:90]: We already have a [documentation for Uninstallation](https://openebs.io/docs/user-guides/uninstall). May be we can add it as an info/note here.

Comment (tiagolobocastro) [reply docs/main/quickstart-guide/installation.md:90]: Makes sense to me

Comment (balaharish7) [reply docs/main/quickstart-guide/installation.md:90]: Will document this change in a separate PR.

**Review by niladrih — APPROVED** [review]

File: docs/main/quickstart-guide/installation.md (hunk 1 above)
Comment (balaharish7) [reply docs/main/quickstart-guide/installation.md]: Documented this change in the [#PR 508](https://github.com/openebs/website/pull/508).

**Pull request merged**